flink-libraries/flink-cep/src/test/java/org/apache/flink/cep/operator/CEPMigrationTest.java

```diff
@@ -75,9 +94,15 @@ public void testKeyedCEPFunctionMigration() throws Exception {
 				BasicTypeInfo.INT_TYPE_INFO);
 
 		harness.setup();
-		harness.initializeStateFromLegacyCheckpoint(getResourceFilename(
-			"cep-keyed-savepoint-1.2"));
-//			"cep-keyed-savepoint-1.1"));
+
+		if (from11) {
```

Comment:
I think we should have separate test classes for "1.1 to 1.3" and "1.2 to 1.3". 

I just though about this in the context of https://issues.apache.org/jira/browse/FLINK-5969 where I'm also planning to do it like this. Sorry for bringing this up so late but I think in the end having a bunch of ifs does not scale well.